app/src/main/res/values/styles.xml

```diff
@@ -180,18 +180,32 @@
         <item name="android:windowExitAnimation">@anim/fade_out</item>
     </style>
 
-    <style name="ThemeIndependentMaterialGreyButton" parent="Widget.MaterialComponents.Button.TextButton">
+    <!-- UI button styling -->
+    <style name="NeutralButton" parent="Widget.MaterialComponents.Button.TextButton">
+        <item name="iconTint">@color/button_text_color</item>
```

Comment:
Are you adding new styling for icons in buttons here?

Reply:
My understanding from UX is that the icons should match the text tint :) another thing I will confirm in the style guide before it's formalized.